CMakeLists.txt

```diff
@@ -89,3 +89,4 @@ file(GLOB SOURCE_FILES
         )
 
 add_executable(dash ${SOURCE_FILES})
+
```

Comment:
You need to revert all these misc changes

Reply:
Unnecessary end-lines etc removed.